### PR TITLE
Create EOSNation

### PR DESCRIPTION
**Pull request by ntlfua**

### CODE REVIEW

File: bp/EOSNation [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  },
+  "account_name": "eosnationcom",
+  "public_key": "EOS8PkNNBYU1xnbcjBUNm1mT6N68QiGvCDgPT97rqurLBEjDanSXK",
+  "org": {
+    "name": "EOS Nation",
+    "website": "http://www.eosnation.io",
+    "email": "info@eosnation.io",
+    "branding":{
+      "logo": "logo-EOSNation.png",
+      "cover": "cover-EOSNation.png"
```
Comment (DenisCarriere) [inline bp/EOSNation]: Add cover =>
https://raw.githubusercontent.com/EOS-Nation/Logos/master/eosnation/eosnation-banner-text.png

File: bp/EOSNation [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    "en": "EOS Nation is dedicated to fostering the EOS platform through reliable and efficient block production and community engagement. We believe in a future where life, liberty, and property is protected through decentralization. Our mission is to bring together all members of the EOS community under one project. Accessible to the community in jurisdictions around the world. We strive to share our vision for the future of EOS through education and through education, engagement and support. EOS Nation is focused on trying to ignite leadership in ourselves and others... Expecting greatness, we will discover it! A global community is possible, and we aim to make it a reality."
+  },
+  "account_name": "eosnationcom",
+  "public_key": "EOS8PkNNBYU1xnbcjBUNm1mT6N68QiGvCDgPT97rqurLBEjDanSXK",
+  "org": {
+    "name": "EOS Nation",
+    "website": "http://www.eosnation.io",
+    "email": "info@eosnation.io",
+    "branding":{
+      "logo": "logo-EOSNation.png",
```
Comment (DenisCarriere) [inline bp/EOSNation]: Add logo =>
https://raw.githubusercontent.com/EOS-Nation/Logos/master/eosnation/eosnation-1x1-black.png

File: bp/EOSNation [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    "en": "Building our Future one Block at a Time"
+  },
+  "introduce": {
+    "en": "EOS Nation is dedicated to fostering the EOS platform through reliable and efficient block production and community engagement. We believe in a future where life, liberty, and property is protected through decentralization. Our mission is to bring together all members of the EOS community under one project. Accessible to the community in jurisdictions around the world. We strive to share our vision for the future of EOS through education and through education, engagement and support. EOS Nation is focused on trying to ignite leadership in ourselves and others... Expecting greatness, we will discover it! A global community is possible, and we aim to make it a reality."
+  },
+  "account_name": "eosnationcom",
+  "public_key": "EOS8PkNNBYU1xnbcjBUNm1mT6N68QiGvCDgPT97rqurLBEjDanSXK",
+  "org": {
+    "name": "EOS Nation",
+    "website": "http://www.eosnation.io",
```
Comment (DenisCarriere) [inline bp/EOSNation]: Short url & https => `https://eosnation.io`

File: bp/EOSNation [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    "email": "info@eosnation.io",
+    "branding":{
+      "logo": "logo-EOSNation.png",
+      "cover": "cover-EOSNation.png"
+    },
+    "location": "Montreal, Canada",
```
Comment (DenisCarriere) [inline bp/EOSNation]: Canada

**Review by ntlfua — COMMENTED** [review]
File updated

**Pull request merged**